tests/quality_pragmaitc/test_interpretation.py

```diff
@@ -17,3 +17,22 @@ def test_interpretation():
 
     assert 1 == len(map)
     assert 1 == len(map[context])
+
+    assert 1 == len(map)
+
+
+def test_getQC():
+    interp = Interpretation()
+    context = Context("C1")
+    commonMetrics = CommonMetrics()
+
+    qc1 = QualityConstraint(
+        context, commonMetrics.SECONDS, 15, Comparison.LESS_THAN)
+    qc2 = QualityConstraint(context, commonMetrics.METERS,
+                            100, Comparison.LESS_THAN)
+
+    interp.addQualityConstraint(qc1)
+    interp.addQualityConstraint(qc2)
+
+    assert qc1 in interp.getQualityConstraints(context)
+    assert qc2 in interp.getQualityConstraints(context)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4895731)